Doc/library/sqlite3.rst

```diff
@@ -828,6 +828,12 @@ Exceptions
    disconnect occurs, the data source name is not found, a transaction could
    not be processed, etc.  It is a subclass of :exc:`DatabaseError`.
 
+.. exception:: NotSupportedError
+
+   Exception raised in case a method or database API was used which is not
+   supported by the database, e.g. requesting a .rollback() on a connection
+   that does not support transaction or has transactions turned off.
+   It must be a subclass of :exc:`DatabaseError`.
```

Comment:
It must be -> It is (PEP 249 is basically a specification, and it has already been implemented in the sqlite3 module, so we don't need use "must be".)

Reply:
Fixed